nlpsandboxclient/__main__.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python3
+import click
 
-from nlpsandboxclient.cli import cli
+from .cli import community, evaluate
 
 
-def main():
-    cli.entrypoint()
+@click.group()
```

Comment:
My intention was to have the root `__main__.py ` free of references to the package `click` to centralize the interface code in the *cli/* folder. I would like to stick to this approach unless there is a clear benefit to do otherwise.

Reply:
So the purpose of `__main__.py` is for it to be executable: https://docs.python.org/3/library/__main__.html.   I can definitely add the `click` specific things back into `cli`, but I was not a huge fan of `nlpsandboxclient.cli.cli`.  We can also put `__main__.py` inside of `nlpsandboxclient.cli`, if you want to keep all command line client things within that folder.